util/schema/registry_schema.json

```diff
@@ -96,10 +96,17 @@
       "suggest": false
     },
     "domain": {
-      "level": "warning",
-      "description": "'domain' is a recommended field. The value is the scientific topic area addressed by the ontology.",
+      "level": "error",
+      "description": "'domain' is a mandatory field. The value is the scientific topic area addressed by the ontology.",
```

Comment:
Consider converting to an enum of allowed values to make sure that no garbage sneaks in

Reply:
Ah you beat me to it!

Reply:
Thanks done that now!